openedx/core/djangoapps/user_api/management/commands/bulk_user_org_email_optout.py

```diff
@@ -0,0 +1,122 @@
+"""
+One-off script to opt-out users for email from orgs.
+
+Input: A CSV file with a user_id,org pair per line. For example:
+
+1962921,FooX
+5506350,BarX
+5709986,FooX
+
+Opts-out every specified user/org combo rows from email by setting the 'email-optin' tag to 'False'.
+"""
+
+from __future__ import print_function
+import csv
+import logging
+import time
+
+from django.db import connections
+from django.core.management.base import BaseCommand, CommandError
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Implementation of the bulk_user_org_email_optout command.
+    """
+    # Default number of user/org opt-outs to perform in each DB transaction.
+    DEFAULT_CHUNK_SIZE = 1000
+
+    # Default number of seconds to sleep between chunked user/org email opt-outs.
+    DEFAULT_SLEEP_BETWEEN_DELETES = 0.0
```

Comment:
It seems weird to have a default that will fail validation on an arg that's not required

Reply:
looks like the validation only fails on negative values, so this seems fine.

Reply:
Ah yep, it's `chunk_size` that's `<=`. Nothing to see here.